float64[] joint_command
string[] joint_name
uint8 mode
uint8 POSITION=1
uint8 TORQUE=2
